config/maintainers.json

```
@@ -50,6 +50,16 @@
       "link_url": "https://johnruble.net/",
       "avatar_url": "https://avatars1.githubusercontent.com/u/164652",
       "bio": "I'm a generalist developer that was introduced to F# on a project at work—lucky, I know!—and it soon became one of my favorite languages. I used Exercism intermittently for a few years, then got more involved contributing to this track during Hacktoberfest."
+    },
+    {
+      "github_username": "umaranis",
+      "alumnus": false,
+      "show_on_website": true,
```

Could you set this to `false`?

```suggestion
      "show_on_website": false,
```